src/main/java/com/pusher/client/channel/PusherEvent.java

```diff
@@ -0,0 +1,47 @@
+package com.pusher.client.channel;
+
+import java.util.Map;
+
+public class PusherEvent {
+    private Map<String, Object> eventData;
+
+    /**
+     * PusherEvent constructor that takes a hashmap.
+     * @param eventData hashmap(string, string)
+     */
+    public PusherEvent(Map<String, Object> eventData) {
+        this.eventData = eventData;
+    }
+    /**
+     * getProperty returns the value associated with the key, or null.
+     * @param key - the key you wish to get
+     * @return value (string) - the value as a string.
+     */
+    public Object getProperty(String key) {
+        return eventData.get(key);
+    }
+
+
+    /**
+     * returns the userId associated with this event.
+     * @return the userID string: https://pusher.com/docs/channels/using_channels/events#user-id-in-client-events,
+     * or null if the customer if the event is not a client event.
+     */
+    public Object getUserId() {
+        return eventData.get("user_id");
+    }
```

Comment:
I think this should be formatted consistently with the methods below it.

Reply:
I think you missed this.

Reply:
oh, i thought you meant the different thing. Changed now.